monitor_packages/mysql/README.md

```diff
@@ -0,0 +1,18 @@
+# sumologic-mysql-monitors
+
+This Terraform script configures pre-packaged Sumo Logic Monitors for MySQL using Terraform modules.
+For installation and configuration, please look at Sumo Logic MySQL help document.
```

Comment:
Add the help page link. It is going to be this link, if we are updating the existing app and doc pages: https://help.sumologic.com/07Sumo-Logic-Apps/12Databases/MySQL/MySQL-App-Dashboards

Reply:
done